packages/react-liveness/src/components/FaceLivenessDetector/hooks/useMediaStreamInVideo.ts

```diff
@@ -11,6 +11,10 @@ export function useMediaStreamInVideo(
   stream: MediaStream,
   videoConstraints: MediaTrackConstraints
 ): UseMediaStreamInVideo {
+  if (!videoConstraints) {
```

Comment:
Is it even possible or `videoConstraints` to be `undefined` now? https://github.com/aws-amplify/amplify-ui/pull/4314/files#diff-49f4d37e97030928199c54c5f8be21e23f8cb9565e90c1cc0845e6c3ad7a7fe9R71

Reply:
It shouldn't be possible but I still added this just in case, the only reason this wasn't getting caught before was we were using `videoConstraints!` at caller side. Happy to remove but was just covering bases

Reply:
Where are we using `videoConstraints!`?

Reply:
It's used when getting the video media recorder object - https://github.com/aws-amplify/amplify-ui/blob/fix-undefined-media/packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts#L878

useMediaStreamInVideo is used to get information about the current media track and pass that to the video html component in the front end here https://github.com/aws-amplify/amplify-ui/blob/fix-undefined-media/packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx#L103